README.md

```diff
@@ -21,6 +21,17 @@ aws cloudformation create-stack \
   --parameters <(cat config.json)
 ```
 
+
+Alternately, if you prefer to use this repo, clone it and run the following command to set up things locally and create a remote stack.
+```bash
+make && make create-stack
+
+# If you need to specify a profile:
```

Comment:
I'd be tempted to just replace this with something like:

"You can use any of the `AWS...` environment variables that the aws-cli supports."